app/src/main/java/de/westnordost/streetcomplete/quests/drinking_water/AddDrinkingWater.kt

```diff
@@ -20,6 +20,9 @@ class AddDrinkingWater : OsmFilterQuestType<DrinkingWater>() {
         and access !~ private|no and indoor != yes
         and !drinking_water and !drinking_water:legal and amenity != drinking_water
         and (!seasonal or seasonal = no)
+        and (!disused or disused = no)
+        and (!historic or historic = no)
```

Comment:
For start, thanks for a PR!

`historic=yes` does not mean that say water well is inactive - only that it is old/has historic value/etc. Can you remove it?

Skipping `disused=yes` makes sense, even if such tagging is a really bad idea.

Not sure about `ruins=yes` - there could be ruined working objects... And none of examples from #4928 have it. Not sure here.

Reply:
I agree that lifecycle prefixes would make far more sense, but that's not what I'm trying to change here. Instead, I'd like to suggest that SC should not ask a question if it doesn't apply. :)

I'll remove the `historic=yes` part.

I forgot to include ruins=yes in the overpass query, but there are matches, for example:
- https://www.openstreetmap.org/node/2624118790
- https://www.openstreetmap.org/way/967657623
- https://www.openstreetmap.org/node/2689111638 (only historic and ruins, but not "disused")

So I'm in favor of keeping the condition in there.

Reply:
By the way, is there a reason why StreetComplete only considers this quest for water-well *nodes*, but not ways?

Reply:
> By the way, is there a reason why StreetComplete only considers this quest for water-well nodes, but not ways?

Probably not. I think it should be asked on ways too.